src/main/java/io/cdap/plugin/http/source/common/http/OAuthUtil.java

```diff
@@ -54,5 +68,54 @@ public static String getAccessTokenByRefreshToken(CloseableHttpClient httpclient
     JsonElement jsonElement = JSONUtil.toJsonObject(responseString).get("access_token");
     return jsonElement.getAsString();
   }
+
+  public static String getAccessTokenByServiceAccount(BaseHttpSourceConfig config) throws IOException {
+    GoogleCredentials credential;
+    Integer expiryLength = (config.getJwtTokenExpiryLength() != null) ? config.getJwtTokenExpiryLength() : 3600;
```

Comment:
I think this var a can be `int`

Reply:
The reason I use Integer is because it can be compared with null. I can't do that with int

Reply:
I understand for config, but I don't think you need to do it for `expiryLength` var